src/App.tsx

```diff
@@ -1,10 +1,57 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/Todolist/TodoList';
+import { TodosFilter } from './components/TodosFilter/TodosFilter';
```

Comment:
I think it will be better when you add index.ts or index.tsx export file with `export * from './TodoList`...